app/views/content/subjects/maths/_article.html.erb

```diff
@@ -0,0 +1,208 @@
+<article>
+  <section class="statement">
+    <div>
+      <span class="pink xlarge bold">Maths is&hellip;</span>
+    </div>
+
+    <div>
+      <span class="pink medium bold">
+        <ul class="slashed-list">
+          <li>going into space</li>
+          <li>getting a mortgage</li>
+          <li>predicting the future</li>
+        </ul>
+      </span>
+    </div>
+  </section>
+
+  <section>
+    <h2>The language of everything</h2>
+
+    <p>
+      Maths is the language we use to find patterns and understand the world
+      around us. It has provided the solution to some of history's most
+      intriguing problems.
+    </p>
+
+    <p>
+      Maths is essential to everyday life, critical to science, technology and
+      engineering as well as being fundamental for most jobs. 
+    </p>
+
+    <p>
+      Maths has a critical role to play in the future of the UK economy. There is
+      significant evidence that adults with numeracy skills earn higher wages and
+      are more likely to be in employment.
+    </p>
+  </section>
+
+  <%= render SubjectSpecific::TextBlockWithTwoImagesComponent.new(
+    heading: "Maths is for everyone",
+    colour: "pink",
+    image_1_path: "media/images/content/hero-images/0017.jpg",
+    image_1_alt: "A teacher sitting with a student helping them with a maths problem",
+    image_2_path: "media/images/content/hero-images/0004.jpg",
+    image_2_alt: "A maths teacher standing at the front of a full classroom delivering a lesson",
+  ) do %>
+
+    <p>
+      If maths is your passion, you know that it is not just learning formulas or
+      equations. It is a way of thinking, and problem solving, that’s important
+      in every walk of life.
+    </p>
+
+    <p>
+      An exciting career spent passing your passion on to the next generation
+      could be a career for you.
+    </p>
+  <% end %>
+
+  <section>
+    <figure>
+      <blockquote class="pink-quotes">
+        <p class="bold">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+          tempor incididunt ut labore et dolore disputandum putant. Sed ut omittam
+          pericula, labores, dolorem etiam, quem optimus quisque pro patria et pro
+          suis.
+        </p>
+      </blockquote>
+
+      <figcaption>Quoniam Nacti</figcaption>
+    </figure>
+  </section>
```

Comment:
@EllieNodder - I added a placeholder for a quote, please feel free to make an edit if you find one :)